core/crypto/crypto.go

```diff
@@ -49,13 +49,13 @@ type Client interface {
 	Node
 
 	// NewChaincodeDeployTransaction is used to deploy chaincode.
-	NewChaincodeDeployTransaction(chaincodeDeploymentSpec *obc.ChaincodeDeploymentSpec, uuid string) (*obc.Transaction, error)
+	NewChaincodeDeployTransaction(chaincodeDeploymentSpec *obc.ChaincodeDeploymentSpec, attributes map[string]string, uuid string) (*obc.Transaction, error)
 
 	// NewChaincodeExecute is used to execute chaincode's functions.
-	NewChaincodeExecute(chaincodeInvocation *obc.ChaincodeInvocationSpec, uuid string) (*obc.Transaction, error)
+	NewChaincodeExecute(chaincodeInvocation *obc.ChaincodeInvocationSpec, attributes map[string]string, uuid string) (*obc.Transaction, error)
```

Comment:
Shall we move attributes to the *Spec in order to avoid breaking retro-compatibility?